### PR TITLE
Trello-590 Use `File` Buffer instead of `Memory` Buffer for Fluentd

### PR DESCRIPTION
**Pull request by bmcelvee**
https://trello.com/c/XpreI533/509-5-use-file-buffer-instead-of-memory-buffer-for-fluentdloggingepic-ois-agl-perf

https://trello.com/c/IzxgZQcL/590-document-use-file-buffer-instead-of-memory-buffer-for-fluentdloggingepic-ois-agl-perf

### CODE REVIEW

**Comment by bmcelvee** [comment]
@nhosoi I saved the content we worked on earlier this month, but to be safe would you mind reviewing again. Thanks!

**Comment by bmcelvee** [comment]
@nhosoi PTAL 

@openshift/team-documentation Please peer review 😸

File: admin_guide/overcommit.adoc [hunk 1]
```diff
@@ -45,67 +45,42 @@ lowest quality of service.
 [[tune-buffer-chunk-limit]]
 === Tune Buffer Chunk Limit
 
-If Fluentd logger is unable to keep up with a high number of logs, you will need
-to increase the compute resource values.
+If Fluentd logger is unable to keep up with a high number of logs, it will need
```
Comment (mburke5678) [inline admin_guide/overcommit.adoc]: s/If Fluentd/If the Fluentd

File: admin_guide/overcommit.adoc [hunk 2]
```diff
@@ -45,67 +45,42 @@ lowest quality of service.
 [[tune-buffer-chunk-limit]]
 === Tune Buffer Chunk Limit
```
Comment (mburke5678) [inline admin_guide/overcommit.adoc]: Is Tune a verb here? As in you are tuning the buffer chunk limit? If so, maybe make it _Tuning the Buffer Chunk Limit_?

File: admin_guide/overcommit.adoc [hunk 3]
```diff
@@ -45,67 +45,42 @@ lowest quality of service.
...
 
-If Fluentd logger is unable to keep up with a high number of logs, you will need
-to increase the compute resource values.
+If Fluentd logger is unable to keep up with a high number of logs, it will need
+to switch to file buffering to reduce memory usage and prevent data loss.
```
Comment (mburke5678) [inline admin_guide/overcommit.adoc]: Does the fluentd logger make this switch on its own? Using the values below that the user configured?

File: admin_guide/overcommit.adoc [hunk 4]
```diff
@@ -45,67 +45,42 @@ lowest quality of service.
...
-
-By default, `buffer_chunk_size` is 1 MB.
-
-The following steps allow you to adjust the available resources.
-
-. Edit the daemonset of Fluentd:
-+
-----
-$ oc edit daemonset logging-fluentd
+On the Fluentd and Mux pods, permanent volume `/var/lib/fluentd` should be
```
Comment (mburke5678) [inline admin_guide/overcommit.adoc]: What's a _Mux_ pod? New term? I did a search in Atom of the files and got no results.

**Comment by mburke5678** [comment]
@bmcelvee Couple of comment

**Comment by bmcelvee** [comment]
@nhosoi I want to make sure we have all of the content up to date in the documentation. Please review when you have a moment. 😸

**Pull request closed without merging**